test/token/ERC1155/extensions/ERC1155Supply.test.js

```diff
@@ -99,9 +109,19 @@ contract('ERC1155Supply', function (accounts) {
       });
 
       it('totalSupply', async function () {
-        expect(await this.token.totalSupply(firstTokenId)).to.be.bignumber.equal('0');
-        expect(await this.token.totalSupply(secondTokenId)).to.be.bignumber.equal('0');
+        expect(await this.token.methods['totalSupply(uint256)'](firstTokenId)).to.be.bignumber.equal('0');
+        expect(await this.token.methods['totalSupply(uint256)'](secondTokenId)).to.be.bignumber.equal('0');
+        expect(await this.token.methods['totalSupply()']()).to.be.bignumber.equal('0');
+      });
+    });
+  });
+
+  context('total supply of token', function () {
+    it('totalSupply unaffected by no-op', async function () {
+      this.token.safeTransferFrom(ZERO_ADDRESS, ZERO_ADDRESS, firstTokenId, firstTokenAmount, '0x', {
+        from: ZERO_ADDRESS,
       });
+      expect(await this.token.methods['totalSupply()']()).to.be.bignumber.equal('0');
```

Comment:
Is there a reason not to check `totalSupply(firstTokenId)` as well?

Reply:
Since Is checked multiple times before this test I did not think we needed it, but do you think we should include it for a no-op test?

Reply:
Yes, `totalSupply(uint256)` is not checked for no-op transfers from zero to zero. I added the test.